rollup.config.js

```diff
@@ -41,6 +42,7 @@ export default [
       resolve(),
       replace({
         'PACKAGE_VERSION_INJECTED_DURING_BUILD': version,
+        preventAssignment: true
```

Comment:
what does this do?

Reply:
There is a warning for replace plugin:
@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to true, as the next major version will default this option to true.

See here what this does: https://github.com/rollup/plugins/tree/master/packages/replace#preventassignment